### ISSUE TITLE
Are created events seriazable?

### ISSUE THREAD

**Issue by LaChope** [kbss-cvut/fta-fmea-ui#290]
@blcham @kostobog After creating the event, what do we do with it? For now I guess it will be an internal (?global) state in the client, but it will need to be sent to the server later I assume?

![Image](https://github.com/kbss-cvut/fta-fmea-ui/assets/57750426/f75a8087-a5c0-41b2-9115-002f1d50db85)

**Comment by blcham** [comment]
After talking to @kostobog, i understood (but NOT SURE), that you should use API to create FTA tree (not providing IRI of the event), and then you need to modify the following method in a similar way:

![image](https://github.com/kbss-cvut/fta-fmea-ui/assets/3962237/47955124-dcba-459b-acf4-5e1077bb5b0d)


Here is original source code:
```
const rootEvent= eventFromHookFormValues(values);
const faultTree = {
    name: values.faultTreeName,
    manifestingEvent: rootEvent,
} as FaultTree;
```

Here is new source code (most-likely does not compile):
```
const supertype = eventFromHookFormValues(values);
    const rootEvent = {
      name: supertype.name,
      supertypes: [supertype]
    } as FaultEvent;
    const faultTree = {
      name: values.faultTreeName,
      manifestingEvent: rootEvent,
    } as FaultTree;
```

**Comment by LaChope** [comment]
Closing as implemented with [590efe6](https://github.com/kbss-cvut/fta-fmea-ui/pull/289/commits/590efe64c815db378db65f0bcb26e9a108e18b39) and will be continued with https://github.com/kbss-cvut/fta-fmea-ui/issues/291
